docs/zk-containers.md

```diff
@@ -2,6 +2,8 @@
 
 We found that developing ZK provers with ICICLE gives developers the ability to scale ZK provers across many machines and many GPUs. To make this possible we developed the ZK Container.
 
+## What is a ZK Container?
```

Comment:
```suggestion
## What is a ZKContainer™️?
```

Reply:
If we choose to use this, i'd suggest using the same term across the following paragraphs